docs/release-notes/eclair-v0.11.0.md

```diff
@@ -0,0 +1,208 @@
+Every node advertizes the rates at which they sell their liquidity, and buyers connect to sellers that offer interesting rates.
+
+The liquidity ads specification is still under review and will likely change.
+This feature isn't meant to be used on mainnet yet and is thus disabled by default.
+
+### Update minimal version of Bitcoin Core
+
+With this release, eclair requires using Bitcoin Core 27.2.
+Newer versions of Bitcoin Core may be used, but have not been extensively tested.
+
+This version introduces a new coin selection algorithm called [CoinGrinder](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-27.0.md#wallet) that will reduce on-chain transaction costs when feerates are high.
+
+To enable CoinGrinder at all fee rates and prevent the automatic consolidation of UTXOs, add the following line to your `bitcoin.conf` file:
+
+```conf
+consolidatefeerate=0
+```
+
+### Incoming obsolete channels will be rejected
+
+Eclair will not allow remote peers to open new `static_remote_key` channels. These channels are obsolete, node operators should use `option_anchors` channels now.
+Existing `static_remote_key` channels will continue to work. You can override this behaviour by setting `eclair.channel.accept-incoming-static-remote-key-channels` to true.
+
+Eclair will not allow remote peers to open new obsolete channels that do not support `option_static_remotekey`.
+
+### HTLC endorsement for channel jamming
+
+Channel jamming mitigations have been under active research for the past year, leading to a proposal based on [local per-node reputation](https://github.com/lightning/bolts/pull/1071).
+A first step towards evaluating the reputation algorithms is to relay an "endorsement" field on outgoing payments, as detailed in [bLIP 4](https://github.com/lightning/blips/pull/27).
+Eclair implements bLIP 4 and will relay endorsement signals, which will allow network-wide experimentation of reputation algorithms in the future.
+
+See #2884 for more details.
+
+### On-the-fly funding
+
+Payments sent to mobile wallets often fail because the recipient doesn't have enough inbound liquidity to receive it.
+This release contains an implementation of our [proposed protocol](https://github.com/lightning/blips/pull/36) to negotiate an on-the-fly liquidity purchase when receiving such payments.
+This protocol builds on top of experimental BOLT features such as splicing and liquidity ads and is used in production in the latest version of [Phoenix](https://phoenix.acinq.co/).
+
+This isn't meant to be activated by default on eclair nodes, but if you want to run an LSP for mobile wallets, you may be interested in this feature.
+This is still experimental at this point and is waiting for other implementations, so use at your own risk!
+
+### API changes
+
+- `channelstats` now takes optional parameters `--count` and `--skip` to control pagination. By default, it will return first 10 entries. (#2890)
```

Comment:
```suggestion
- `channelstats` now takes optional parameters `--count` and `--skip` to control pagination. By default, it will return the first 10 entries. (#2890)
```